processing/Dockerfile

```diff
@@ -1,18 +1,27 @@
 FROM python:3.7-stretch
 
-ARG LOCAL_USER_ID
+#RUN useradd -rm -d /home/gazette -s /bin/bash -g root -G sudo -u 1000 gazette
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && apt-get install --no-install-recommends -y \
+    poppler-utils \
+    postgresql-client \
+    wait-for-it \ 
+    default-jre \ 
+    && rm -rf /var/lib/apt/lists/*
+
+#USER gazette
 
 WORKDIR /mnt/code
 
-COPY . /mnt/code
+COPY . . 
+
+ADD http://archive.apache.org/dist/tika/tika-app-1.24.jar tika-app.jar
+
+RUN pip install --no-cache-dir -r requirements.txt 
 
-# install Apache Tika
-ADD http://archive.apache.org/dist/tika/tika-app-1.24.jar /tika-app.jar
+#RUN chmod 755 tika-app.jar 
 
-RUN adduser --system -u ${LOCAL_USER_ID:-1000} gazette \
-	&& apt-get update \
-	&& apt-get -y install poppler-utils postgresql-client wait-for-it default-jre \
-	&& pip install --no-cache-dir -r requirements.txt \
-	&& chmod 755 /tika-app.jar
+WORKDIR /mnt/code/data_collection
```

Comment:
Please, remove this line. Let's continue using the base `/mnt/code` as workdir. I do not see advantages of that. If we add some other directory in the project or use the same image for something else, we will need to remember that we are inside the `mnt/code/data_collection`. Remember to change the command above, where you remove the `cd data_collection`. ;)